webview_panels/src/modules/documentationEditor/components/tests/DisplayTestDetails.tsx

```diff
@@ -0,0 +1,307 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as Yup from "yup";
+import { useForm } from "react-hook-form";
+import { executeRequestInSync } from "@modules/app/requestExecutor";
+import {
+  DbtGenericTests,
+  DBTModelTest,
+  TestMetadataAcceptedValuesKwArgs,
+  TestMetadataRelationshipsKwArgs,
+} from "@modules/documentationEditor/state/types";
+import { panelLogger } from "@modules/logger";
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardTitle,
+  CodeBlock,
+  IconButton,
+  ListGroup,
+  ListGroupItem,
+  Stack,
+  Tag,
+} from "@uicore";
+import { useMemo, useState } from "react";
+import AcceptedValues from "./forms/AcceptedValues";
+import Relationships from "./forms/Relationships";
+import { SaveRequest } from "./types";
+import useTestFormSave, { TestOperation } from "./hooks/useTestFormSave";
+import classes from "../../styles.module.scss";
+import { DeleteIcon, EditIcon } from "@assets/icons";
+
+const schema = Yup.object({
+  to: Yup.string().optional(),
+  field: Yup.string().optional(),
+  accepted_values: Yup.array().of(Yup.string().required()).optional(),
+}).required();
+
+interface Props {
+  onClose: () => void;
+  test: DBTModelTest;
+  column: string;
+}
+
+const DisplayTestDetails = ({ onClose, test, column }: Props): JSX.Element => {
+  const { control, handleSubmit, setValue, watch } = useForm<SaveRequest>({
+    resolver: yupResolver(schema),
+  });
+
+  const { isSaving, handleSave } = useTestFormSave();
+
+  const [testCode, setTestCode] = useState("");
+  const [isInEditMode, setIsInEditMode] = useState(false);
+
+  const isEditableTest =
+    test.test_metadata?.name !== DbtGenericTests.NOT_NULL &&
+    test.test_metadata?.name !== DbtGenericTests.UNIQUE;
+
+  const handleDelete = () => {
+    panelLogger.info("delete test", test);
```

Comment:
will pollute log

Reply:
these are just console logs in webview only. it will not show in terminal